### DISCUSSION TITLE
VPS hosting with iPXE iso via http

### DISCUSSION THREAD

**Discussion by Nick-1231** [ipxe/ipxe#784] [Q&A]
Hello!

I rented a small VPS server and I want to boot Ubuntu, but on my hosting I have only one option - to boot iso iPXE via HTTP.

I need Ubuntu 22.10 Desktop (x64) iPXE iso (via http) and Ubuntu 22.10 server (x64) iPXE iso (via http).

**P.S. I am very, very need help! If anybody know, please write http link to both iPXE iso…**

**Comment by NiKiZe** [comment]
You generally can not boot Isos.
You could however probably boot into a script which does all the needed things. netboot.xyz might have what you need for a easy fix.

**Reply by Nick-1231** [reply]
@NiKiZe
First, thank you for reply.
"You generally can not boot Isos" - VMmanager have this feature (see link below (mounting-an-iso-image))

I try this link: https://boot.netboot.xyz/ipxe/netboot.xyz.iso but «VMmanager» said, that «ISO check is failed»!

Error #5156:
**«ISO check is failed»**

VMmanager Documentation link:
https://docs.ispsystem.com/v6/virtual-machines/mounting-an-iso-image

I can`t find a proper Ubuntu 22.10 iPXE iso…

**P.S. May be someone have right Ubuntu 22.10 iPXE iso, which I can download via http in «VMmanager»??? Help, please…**
